lib/plug/cowboy/handler.ex

```diff
@@ -51,15 +51,19 @@ defmodule Plug.Cowboy.Handler do
 
   defp exit_on_error(:error, value, stack, call) do
```

Comment:
Unfortunately we should not change the exit format. The format we use above is standardized by Erlang/OTP's `proc_lib` module. So changing it can break Logger and other monitoring tools in other ways.

Unfortunately I am not sure if there is something we can do here, given Cowboy logs exceptions from another process.

Reply:
Oh, I see... :face_with_diagonal_mouth:  

Thanks for quick feedback.

Reply:
> Unfortunately I am not sure if there is something we can do here, given Cowboy logs exceptions from another process.

Yeah, tricky.